tests/components/test_snips.py

```diff
@@ -1,45 +1,227 @@
-"""Test the Snips component."""
-import asyncio
-
-from homeassistant.bootstrap import async_setup_component
-from tests.common import async_fire_mqtt_message, async_mock_intent
-
-EXAMPLE_MSG = """
-{
-    "input": "turn the lights green",
-    "intent": {
-        "intentName": "Lights",
-        "probability": 1
-    },
-    "slots": [
+"""The tests for the Dialogflow component."""
+import unittest
+
+from homeassistant.core import callback
+from homeassistant.components import snips
+from homeassistant.setup import async_setup_component, setup_component
+from tests.common import (get_test_home_assistant, fire_mqtt_message,
+                          mock_mqtt_component, mock_component, mock_service)
+
+
+class TestSnips(unittest.TestCase):
```

Comment:
I am not sure why you changed the test format from independent tests to be based on `unittest.TestCase` ? The independent tests format is preferred.

Reply:
Gotcha, was basing it on the dialogflow component. In general, just running through the tests it *seemed* like unittest was more prevalent on recent tests so I went with that. But happy to redo it.

Reply:
Took a run at it but couldn't figure it out in the time I had.